circbuf_test.go

```diff
@@ -196,3 +196,46 @@ func TestBuffer_Reset(t *testing.T) {
 		t.Fatalf("bad: %v", string(buf.Bytes()))
 	}
 }
+
+func TestBuffer_WriteByte(t *testing.T) {
+	inp := []byte("hello world")
+
+	buf, err := NewBuffer(3)
+	if err != nil {
+		t.Fatalf("err: %v", err)
+	}
+
+	for _, b := range inp {
+		buf.WriteByte(b)
+	}
+
+	expect := []byte("rld")
+	actual := buf.Bytes()
+	if !bytes.Equal(actual, expect) {
+		t.Fatalf("bad: %v", actual)
+	}
+}
+
+func TestBuffer_ReadByte(t *testing.T) {
+	inp := []byte("hello world")
+
+	buf, err := NewBuffer(int64(len(inp)))
+	if err != nil {
+		t.Fatalf("err: %v", err)
```

Comment:
I meant this kind of error. Make it more informative. `(failed to allocate new buffer %v, err)` something like that.

Reply:
@zozo123 I'm just following the style of the rest of the document -- if you look around `circbuf_test.go` you'll see it's `err:` and `bad:` everywhere.